mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -221,9 +212,8 @@ void AppearanceCallback(IConVar *var, const char *pOldValue, float flOldValue)
 static CMomentumPlayer *s_pPlayer = nullptr;
 
 CMomentumPlayer::CMomentumPlayer()
-    : m_duckUntilOnGround(false), m_flStamina(0.0f),
-      m_flLastVelocity(0.0f), m_nPerfectSyncTicks(0), m_nStrafeTicks(0), m_nAccelTicks(0),
-      m_nPrevButtons(0), m_flTweenVelValue(1.0f), m_bInAirDueToJump(false), m_iProgressNumber(-1)
+    : m_duckUntilOnGround(false), m_flStamina(0.0f), m_flLastVelocity(0.0f), m_nPerfectSyncTicks(0), m_nStrafeTicks(0),
+      m_nAccelTicks(0), m_nPrevButtons(0), m_flTweenVelValue(1.0f), m_bInAirDueToJump(false), m_iProgressNumber(-1)
```

Comment:
Honestly, if you can just undo all of the formatting changes and only commit the actual code changes you need for this branch that would be splendid.